packages/matrix/tests/live-cards.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { test, expect } from '@playwright/test';
+import { writeJSONSync, writeFileSync } from 'fs-extra';
+import { join } from 'path';
+import {
+  synapseStart,
+  synapseStop,
+  type SynapseInstance,
+  registerUser,
+} from '../docker/synapse';
+import {
+  startServer as startRealmServer,
+  type IsolatedRealmServer,
+  appURL,
+} from '../helpers/isolated-realm-server';
+import {
+  clearLocalStorage,
+  createRealm,
+  login,
+  registerRealmUsers,
+  showAllCards,
+} from '../helpers';
+
+test.describe('Live Cards', () => {
```

Comment:
i'm encountering enough live card tests in matrix that i felt it made sense to make a test module just for this using the isolated realm. (it's truly pleasant to test SSE in matrix since there is none of the `expectEvents()` nonsense :stuck_out_tongue_closed_eyes: )

Reply:
could the majority or all of the tests that use that be migrated to Matrix tests instead?

Reply:
It does use the expensive isolated realm, so there is a real cost. But I tried to use a bunch of SSE assertions in the same test. It might not be a bad idea, though, to port over what makes sense. I think these SSE tests are much easier to read and maintain IMHO